homeassistant/components/light/limitlessled.py

```diff
@@ -40,19 +41,19 @@
 
 EFFECT_NIGHT = 'night'
 
-RGB_BOUNDARY = 40
+MIN_SATURATION = 40
```

Comment:
This should probably be lower (like 10?) because its scale now has a maximum of 100 and not 255.

(Maybe it should even be zero but that is a different issue.)

Reply:
Great catch!